api/overview/azure/batch.md

```diff
@@ -29,12 +37,44 @@ Install-Package Azure.Batch
 dotnet add package Azure.Batch
 ```
 
+### Example
+
+The following example uses the client SDK to create a job to run in Azure Batch.
+
+```csharp
+/*
+using Microsoft.Azure.Batch.Auth;
+using Microsoft.Azure.Batch.Common;

```

Comment:
For the sample we only need the Microsoft.Azure.Batch namespace.
Microsoft.Azure.Batch.Auth if you add BatchSharedKeyCredentials. See comment below.